File: GM_ClientSide/src/app/models/indexMessage.model.ts
export class IndexMessageModel {
    id: string;
    conversationId: string;
    senderId: string;
    seenBy: string;
    content: string;
    type: string;
    createdOn: Date;
    receiver: string;

    public constructor(conversationId: string,
        senderId: string,
        seenBy: string,
        content: string,
        type: string,
        receiver: string) {
        this.conversationId = conversationId;
        this.senderId = senderId;
        this.seenBy = seenBy;
        this.content = content;
        this.type = type;
        this.receiver = receiver;
    }
}

File: Backend/GrooveMessengerDAL/Models/Message/IndexMessageModel.cs
using System;

namespace GrooveMessengerDAL.Models.Message
{
    public class IndexMessageModel
    {
        public Guid Id { get; set; }

        public DateTime CreatedOn { get; set; }

        public Guid ConversationId { get; set; }

        public string SenderId { get; set; }

        public string SeenBy { get; set; }

        public string Content { get; set; }

        public string Type { get; set; }
    }
}